### DISCUSSION TITLE
A permalink without the function

### DISCUSSION THREAD

**Discussion by Sylvain1978** [pamelafox/recursive-visualizations#25] [Ideas]
Hi, I often use recursionvisualizer  as a support to explain recursion to my students;
It would be nice if it would be possible to get a permalink for the recursion tree without the function so as to ask them to find the function from the recursion tree. Or the possibility to generate a .gif would be ok .
Thank you

**Comment by pamelafox** [comment]
Sure, good idea, you can add a &hide_function=true to the end of a permalink URL now. Of course, this is only useful when youre presenting, since its easy to then show the function.
